application/models/models.py

```diff
@@ -92,11 +92,10 @@ class ClassificationModel(Base):
     This models the 'classifier' table which keeps tracks tags against an instrument
     """
     __tablename__ = 'classification'
-    classifications = ('LEGAL_STATUS', 'INDUSTRY', 'SIZE', 'GEOGRAPHY')
     id = Column(Integer, primary_key=True)
     instrument_id = Column(Integer, ForeignKey('instrument.id'))
     instrument = relationship('InstrumentModel', back_populates='classifications')
-    kind = Column(Enum('LEGAL_STATUS', 'INDUSTRY', 'SIZE', 'GEOGRAPHY', 'COLLECTION_EXERCISE', 'RU_REF', name='kind'))
+    kind = Column(String(64))
```

Comment:
Your right, this will cause some pain. I have reverted it for the minute as it currently doesn't matter and the business are yet to decide on kind. I will raise another card so not to stop this progressing